apps/web/src/components/milestone-logs/recruitment/Milestone.tsx

```diff
@@ -46,14 +45,10 @@ export const AddMilestones: React.FC<AddMilestoneProps> = ({
     values.job_id = jobId.toString();
     values.added_by = '1';
 
-    try {
-      const {
-        data: { data },
-      } = await addMilestone(applicantId as string, values);
+    const data = await addMilestone(applicantId as string, values);
```

Comment:
applicantId's type is `string | string[] | undefined`. should we change it to `string` or check if it is string here?

Reply:
changed it to string in parent comp so we can change type to just string